client.go

```diff
@@ -1011,3 +1025,32 @@ func (c *TunaSessionClient) IsSessClosed(sessKey string) bool {
 func (c *TunaSessionClient) SetTunaNode(node *types.Node) {
 	c.tunaNode = node
 }
+
+func (c *TunaSessionClient) getCachedPubAddrs(remoteAddr string, sessionID []byte) *PubAddrs {
```

Comment:
This cache only works for the same sessionID. But pubAddrs should be the same regardless of sessionID, so creating new sessions can be a lot faster.